docs.md

```diff
@@ -0,0 +1,3 @@
+* [The rustwasm book](docs/book)
+* [The wasm-bindgen book](docs/wasm-bindgen)
+* [The wasm-pack book](docs/wasm-pack)
```

Comment:
This page is very bare bones and much smaller than it would otherwise want to be before merging